python/lsst/scarlet/lite/io.py

```diff
@@ -118,6 +250,40 @@ class ScarletFactorizedComponentData:
     def shape(self):
         return self.morph.shape
 
+    def to_component(self, observation: Observation) -> FactorizedComponent:
+        """Convert the storage data model into a scarlet FactorizedComponent
+
+        Parameters
+        ----------
+        observation:
```

Comment:
This numpydoc is not valid (there has to be a space before the colon) and the type has to be fully expanded even though it's in the signature... This is the case everywhere.